recipes/qgeoview/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class QGeoViewConan(ConanFile):
+    name = "qgeoview"
+    description = "QGeoView is a Qt / C ++ widget for visualizing geographic data. "
+    license = "LGPL-3.0"
+    topics = ("geo", "geospatial", "qt")
+    homepage = "https://github.com/AmonRaNet/QGeoView"
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        self.options.rm_safe("fPIC")
+        # FIXME: we shouldn't have to force shared in qt, but config file
+        #        generated by conan in qt static is likely broken, or maybe env vars.
+        self.options["qt"].shared = True
+
+    def requirements(self):
+        self.requires("qt/[>=5.15.2 <7]")
```

Comment:
Hi! Thanks for the idea, but QT is not yet one of the allowed libraries to be required with version ranges - We're working towards that future, but understanding the implications for each library is a slow process :) Please pin this for now with the version that makes most sense, usually the latest available one in CCI

Reply:
Hi! Thank you for your attention. I tried to fix qt5 or qt6. Conan v1 pipeline works with qt5 but v2 pipeline works only with qt6.